examples/nordic/nrf5x/src/blinky.zig

```diff
@@ -4,6 +4,14 @@ const board = microzig.board;
 const nrf = microzig.hal;
 const time = nrf.time;
 
+const rtc_overflow_interrupt = nrf.time.rtc_overflow_interrupt;
+
+pub const microzig_options = microzig.Options{
+    .log_level = .debug,
+    .logFn = nrf.uart.log,
+    .interrupts = .{ .RTC0 = .{ .c = rtc_overflow_interrupt } },
```

Comment:
We'd have to make this default. I think to do this we would have to merge the option cpu InterruptOptions with a new chip InterruptOptions